src/apps/ActorSheet.js

```diff
@@ -80,4 +82,44 @@ export default class ActorSheet extends SvelteApplication {
       ...startingEquipment
     ]);
   }
+
+  async _onDropCulture(item) {
+    if (item?.type !== 'culture') throw Error('_onDropCulture() must be called with a culture type item.');
+
+    if (this.actor.type !== 'character') {
+      ui.notifications.warn('Background documents cannot be added to NPCs.');
+      return;
+    }
+
+    // Concat known languages with newly learned, removing duplicates.
+    let known = this.actor.system.proficiencies.languages;
+    known = known.concat(item.system.languages.learn);
+
+    const addLangugaes = item.system.languages.additional;
+    if (addLangugaes > 0) {
+      const dialog = new LanguageSelectDialog({
+        languages: known,
+        disabled: known,
+        /* TODO: Add selectionCount to the LanguageSelect Dialog.
+        This can be done after languages get delocalized. */
```

Comment:
Since we're no longer removing localisation, it seems like this is something that either needs to be removed or addressed.

Reply:
I'm going to attempt to refactor the `LanguageSelect` to be more generic (or at least derived)  for all traits/misc proficiencies (tools, weapons, armor, immunities, resistances, etc). So whatever happens the LanguageSelect is going to get heavily refactored. The PR is specifically focused on culture items. So just having barebones working is fine, the `TODO` is perfectly acceptable so this change doesn't get ignored later on.